routes/rename-project.js

```diff
@@ -0,0 +1,25 @@
+var utils = require("./utils");
+
+module.exports = function(config) {
+  return function(req, res) {
+    var project = JSON.parse(JSON.stringify(req.session.project.meta));
```

Comment:
This is the easiest way to clone an object in js (without the prototype). This will let me make changes to the object without changing the original object